test/scalability/mzcompose.py

```diff
@@ -389,6 +394,18 @@ def create_result_analyzer(_args: argparse.Namespace) -> ResultAnalyzer:
     return DefaultResultAnalyzer(max_deviation_in_percent=0.1)
 
 
+def upload_regressions_to_buildkite(outcome: RegressionOutcome) -> None:
+    if not outcome.has_regressions():
+        return
+
+    file_name = "regressions.csv"
+    file_path = RESULTS_DIR / file_name
+    outcome.raw_regression_data.to_csv(file_path)
+    spawn.runv(
+        ["buildkite-agent", "artifact", "upload", file_name], cwd=RESULTS_DIR
```

Comment:
You could also list the file in the pipeline file, then it will be picked up automatically.

Reply:
I will need to test if this works also when the file is located in some path.